src/main/java/com/ericsson/ei/frontend/RulesController.java

```diff
@@ -0,0 +1,116 @@
+package com.ericsson.ei.frontend;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.protocol.HTTP;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/rules")
+public class RulesController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(RulesController.class);
+
+    @Value("${ei.backendServerHost}")
+    private String backendServerHost;
+
+    @Value("${ei.backendServerPort}")
+    private int backendServerPort;
+
+    @Value("${ei.backendContextPath}")
+    private String backendContextPath;
+
+    @Value("${ei.useSecureHttp}")
+    private boolean useSecureHttp;
+
+    @CrossOrigin
+    @RequestMapping(value = "/rule-check/aggregation", method = RequestMethod.POST)
+    public ResponseEntity<String> postRequests(HttpServletRequest request) {
+
```

Comment:
What special with this POST entry point,,, can't you use the general POST Request Bridge entrypoint?